### PR TITLE
Meaningless values for {model_name}_cpl_dt in nuopc.runconfig 1deg_jra55do_ryf

### PR DESCRIPTION
**Pull request by ezhilsabareesh8**
This PR address [#132](https://github.com/COSIMA/access-om3/issues/132) by setting meaningless values for `{model_name}_cpl_dt` in nuopc.runconfig .

### CODE REVIEW

**Review by dougiesquire — CHANGES_REQUESTED** [review]
Can you remove additional spaces - see https://github.com/COSIMA/MOM6-CICE6/pull/55#pullrequestreview-2034683326

File: nuopc.runconfig [hunk 1]
```diff
@@ -278,19 +278,19 @@ MED_attributes::
...
      end_restart = .false.
      glc_avg_period = yearly
      glc_cpl_dt = 86400
      history_ymd = -999
-     ice_cpl_dt = 3600
-     lnd_cpl_dt = 3600
-     ocn_cpl_dt = 3600
+     ice_cpl_dt =  99999      #not used
+     lnd_cpl_dt =  99999      #not used
+     ocn_cpl_dt =  99999      #not used
```
Comment (aekiss) [inline nuopc.runconfig:289]:
```suggestion
     ice_cpl_dt = 99999      #not used
     lnd_cpl_dt = 99999      #not used
     ocn_cpl_dt = 99999      #not used
```

File: nuopc.runconfig [hunk 2]
```diff
@@ -278,19 +278,19 @@ MED_attributes::
...
      restart_n = 1
      restart_option = nmonths
      restart_ymd = -999
-     rof_cpl_dt = 3600
+     rof_cpl_dt =  99999      #not used
```
Comment (aekiss) [inline nuopc.runconfig:293]:
```suggestion
     rof_cpl_dt = 99999      #not used
```

File: nuopc.runconfig [hunk 3]
```diff
@@ -300,7 +300,7 @@ CLOCK_attributes::
      tprof_n = -999
      tprof_option = never
      tprof_ymd = -999
-     wav_cpl_dt = 3600
+     wav_cpl_dt = 99999       #not used
```
Comment (aekiss) [inline nuopc.runconfig:303]:
```suggestion
     wav_cpl_dt = 99999 #not used
```

File: nuopc.runconfig [hunk 4]
```diff
@@ -278,19 +278,19 @@ MED_attributes::
 ::
 
 CLOCK_attributes::
-     atm_cpl_dt = 3600
+     atm_cpl_dt = 99999       #not used
```
Comment (aekiss) [inline nuopc.runconfig:281]:
```suggestion
     atm_cpl_dt = 99999 #not used
```

**Comment by dougiesquire** [comment]
In the TWG meeting today, we agreed to set `ocn_cpl_dt` to the coupling timestep and all other `<component>_cpl_dt = 99999`. We decided this was the clearest approach that still works with `stop_option = nsteps`. @ezhilsabareesh8 would you be able to update this PR?

File: nuopc.runconfig (hunk 4 above)
Comment (ezhilsabareesh8) [reply nuopc.runconfig]: Resolved in this commit [5899c1c](https://github.com/COSIMA/MOM6-CICE6/pull/56/commits/5899c1c4b8c0c972a5fb526147031e1fea3f5fd5)

File: nuopc.runconfig (hunk 3 above)
Comment (ezhilsabareesh8) [reply nuopc.runconfig:303]: Resolved in this commit [5899c1c](https://github.com/COSIMA/MOM6-CICE6/pull/56/commits/5899c1c4b8c0c972a5fb526147031e1fea3f5fd5)

File: nuopc.runconfig (hunk 1 above)
Comment (ezhilsabareesh8) [reply nuopc.runconfig]: Resolved in this commit [5899c1c](https://github.com/COSIMA/MOM6-CICE6/pull/56/commits/5899c1c4b8c0c972a5fb526147031e1fea3f5fd5)

File: nuopc.runconfig (hunk 4 above)
Comment (ezhilsabareesh8) [reply nuopc.runconfig]: Resolved in this commit [5899c1c](https://github.com/COSIMA/MOM6-CICE6/pull/56/commits/5899c1c4b8c0c972a5fb526147031e1fea3f5fd5)

File: nuopc.runconfig (hunk 2 above)
Comment (ezhilsabareesh8) [reply nuopc.runconfig]: Resolved in this commit [5899c1c](https://github.com/COSIMA/MOM6-CICE6/pull/56/commits/5899c1c4b8c0c972a5fb526147031e1fea3f5fd5)

File: nuopc.runconfig [hunk 5]
```diff
@@ -278,19 +278,19 @@ MED_attributes::
 ::
 
 CLOCK_attributes::
-     atm_cpl_dt = 99999       #not used
+     atm_cpl_dt = 99999 #not used
      calendar = NO_LEAP
      end_restart = .false.
      glc_avg_period = yearly
      glc_cpl_dt = 86400
```
Comment (minghangli-uni) [inline nuopc.runconfig:285]: `glc_cpl_dt = 99999` instead of 86400

Comment (ezhilsabareesh8) [reply nuopc.runconfig]: Updated in this commit 58fc8fc

**Review by dougiesquire — APPROVED** [review]
Thanks @ezhilsabareesh8 et al. LGTM

**Comment by dougiesquire** [comment]
Please squash and merge to clean up the commit history

**Pull request merged**